tests/probes_test.go

```diff
@@ -140,12 +128,30 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 		tcpProbe := createTCPProbe(period, initialSeconds, port)
 		httpProbe := createHTTPProbe(period, initialSeconds, port)
 
-		table.DescribeTable("should not fail the VMI", func(livenessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
-			By("Specifying a VMI with a liveness probe")
-			vmi := createReadyCirrosVMIWithLivenessProbe(virtClient, livenessProbe)
+		table.DescribeTable("should not fail the VMI", func(livenessProbe *v12.Probe, IPFamily v1.IPFamily) {
+
+			if IPFamily == v1.IPv6Protocol {
+				libnet.SkipWhenNotDualStackCluster(virtClient)
+
+				By("Create a support pod which will reply to kubelet's probes ...")
+				probeBackendPod, supportPodCleanupFunc := buildProbeBackendPodSpec(livenessProbe)
+				defer func() {
+					Expect(supportPodCleanupFunc()).To(Succeed(), "The support pod responding to the probes should be cleaned-up at test tear-down.")
+				}()
 
-			By("Starting the server inside the VMI")
-			serverStarter(vmi, 1500)
+				By("Attaching the liveness probe to an external pod server")
+				livenessProbe, err = pointProbeToSupportPod(probeBackendPod, IPFamily, livenessProbe)
+				Expect(err).ToNot(HaveOccurred(), "should attach the backend pod with livness probe")
+
+				By("Specifying a VMI with a readiness probe")
+				vmi = createReadyCirrosVMIWithReadinessProbe(virtClient, livenessProbe)
```

Comment:
This talks about readiness probe while the test is mean for liveness.

Reply:
ooops. Done.